dt-posts/posts.php

```diff
@@ -310,19 +310,19 @@ public static function format_connection_message( $p2p_id, $activity, $action =
             }
         } else if ( $p2p_type === "groups_to_groups"){
             if ($action == "connected to"){
-                $object_note_to = sprintf( esc_html_x( '%s added to child groups', 'group2 added to child groups', 'disciple_tools' ), $from_title );
+                $object_note_to = sprintf( esc_html_x( '%s added to child groups', 'group1 added to child groups', 'disciple_tools' ), $from_title );
```

Comment:
Change not needed.

Reply:
There were 2 strings the same but showed as different because the context was different - hence I changed this context to reflect the other stings' context.

Reply:
I only see one string about adding to child groups

Reply:
Changed back (but not sure why it said group2 instead of group1 like all the rest)
Whatever - no big deal.